taglib/mp4/mp4atom.cpp

```diff
@@ -32,10 +32,10 @@
 
 using namespace TagLib;
 
-const char *const MP4::Atom::containers[11] = {
-    "moov", "udta", "mdia", "meta", "ilst",
-    "stbl", "minf", "moof", "traf", "trak",
-    "stsd"
+constexpr std::array<const char *, 11> MP4::Atom::containers {
```

Comment:
Could this be simplified to `constexpr std::array` without the template arguments?

Reply:
Nope. I tried doing this but GCC complained.

The problem is that this is forward declared in the header. It's not in an anonymous namespace local to the file.

Reply:
I noticed that also and it seems like a candidate to be moved to the cpp file. Something like

```c++
namespace {
constexpr std::array containers {
  "moov", "udta", "mdia", "meta", "ilst",
  "stbl", "minf", "moof", "traf", "trak",
  "stsd"
};
}
```

A quick check didn't show that `containers` was used in any other file so I think that would work.

Reply:
Done.